lib/Parse/ParseDecl.cpp

```diff
@@ -1630,7 +1630,7 @@ parseCompletionHandlerAsyncAttribute(Parser &P, StringRef AttrName,
 
   return new (P.Context) CompletionHandlerAsyncAttr(
       parsedAsyncName.formDeclNameRef(P.Context), nameLoc,
-      handlerIndexLoc.isValid(), handlerIndex, handlerIndexLoc, AtLoc,
+      handlerIndex, handlerIndexLoc, AtLoc,
```

Comment:
Just noticed that `handlerIndex` is uninitialized in the default case, probably worth setting it to something even if it'll end up being overridden in type checking. 0/max?

Reply:
Okay, zero initialized now.